### ISSUE TITLE
Daxko module looks outdated

### ISSUE THREAD

**Issue by ddrozdik** [ymcatwincities/openy#2369]
It seems daxko module(the first version) is outdated already and can be deleted from the distribution. Could you check it?

**Comment by sarah-halby** [comment]
@ddrozdik is this in regards to Activity Finder or just in general? I want to make sure @podarok has the information he needs to move forward.

**Comment by ddrozdik** [comment]
@sarah-halby in general, the module https://github.com/ymcatwincities/openy/tree/9.x-2.x/modules/custom/daxko was created 4 years ago. There are some dependencies to this module however, they are not relevant anymore. 
On the other hand, there is a module https://github.com/ymcatwincities/openy/tree/9.x-2.x/modules/custom/openy_daxko2 that works with API v2 and seems the most used across all Ys. 

I don't have much information to decide that's why we need to investigate if we can safely deprecate it and remove it in future versions.

**Comment by podarok** [comment]
I agree that if we are not using it anymore - it is a good candidate for deprecation and decoupling to a dedicated repository just in case if needed
I guess due to this is a change, functional change, we should be careful and we need to notify the community this is coming in the same way as we do for the media_*, openy_autocomplete_path, and some other modules, described in the upcoming February release as deprecated.
@sarah-halby

**Comment by sarah-halby** [comment]
@ddrozdik can you confirm that your clients are not using this module? I will connect with OneEach, JW Tech, and ImageX. The plan is that the code is not removed and not decoupled, however, it will not be tested in new releases.

**Comment by ddrozdik** [comment]
@sarah-halby yes, I confirm we do not use it across all our clients.

**Comment by podarok** [comment]
```
 The service "openy_programs_search.data_storage" has a dependency on a non-  
  existent service "daxko.client".   
```

**Comment by sarah-halby** [comment]
@podarok should this ticket be closed after you confirm with IMX and YUSA that they are not using this module?
